helm/dagster/values.yaml

```diff
@@ -615,8 +615,27 @@ dagsterDaemon:
 # Extra Manifests: (Optional) Create additional k8s resources within this chart
 ####################################################################################################
 extraManifests:
+## Example 1:
+## Set default container resource requests/limits for the current namespace
+##   * To override these for dagster system containers; edit the resources sections of
+##     this values yaml -  eg: dagit.resources and/or celery.resources
+##   * To override these in solid execution containers; add a @solid(tag=) similar to:
+##      { "dagster-k8s/config": { "container_config": { "resources": {...
+##- apiVersion: v1
```

Comment:
Oops! our linter failed because lines 624-636 are not indented properly. adding a 2-space indent to these lines would fix it